CHANGELOG.md

```diff
@@ -23,7 +23,10 @@ Main (unreleased)
 - `otelcol.exporter.prometheus`: Set `include_scope_info` to `false` by default. You can set 
   it to `true` to preserve previous behavior. (@gouthamve)
 
-- `prometheus.remote_write`: Set `retry_on_http_429` to `true` by default in the `queue_config` block.
+- `prometheus.remote_write` (flow): Set `retry_on_http_429` to `true` by default in the `queue_config` block.
+  You can set it to `false` to preserve previous behavior. (@wildum)
+
+- `remote_write` (static): Override prometheus default value of `retry_on_http_429` in the `queue_config` to `true`.
```

Comment:
Sorry for the many iterations, but I think this could be improved by collating into a single entry in the changelog. Say that the retry on 429 is changed to default to true and that it affects both the static mode's remote_write and flow mode's prometheus.remote_write.